internal/commands/network/create.go

```diff
@@ -25,7 +26,12 @@ var _ commands.Command = &createCommand{}
 // CreateCommand creates the 'network create' command
 func CreateCommand() commands.Command {
 	return &createCommand{
-		BaseCommand: commands.New("create", "Create a network", ""),
+		BaseCommand: commands.New(
+			"create",
+			"Create a network",
+			"upctl network create --name \"My Network\" --zone pl-waw1 --ip-network address=10.0.1.0/24",
```

Comment:
Very minor, but maybe we could use backquotes here to not have to escape the quotes?

Reply:
Done